bin/run_tests.sh

```diff
@@ -4,6 +4,13 @@ set -e
 base=$(pwd)
 failed=0
 
+# run snyk tests locally. Skip in bamboo due to authentication token requirement.
```

Comment:
We should have some docs on the authentication token and how to pass it into this script. Probably fine just adding some comments to this file.

Reply:
added some comments